src/Relation/Nullary/Decidable/Core.agda

```diff
@@ -52,6 +51,15 @@ open Dec public
 pattern yes p =  true because ofʸ  p
 pattern no ¬p = false because ofⁿ ¬p
 
+------------------------------------------------------------------------
+-- decision procedures are Decidable
+
+bool⁺ : ∀ b → Dec (T b)
```

Comment:
Name suggestion: `T?`

Reply:
Well, quite, and then we are precisely (redundantly) duplicating the definition between this module and `Data.Bool.Properties`, and as I've argued above (perhaps not altogether successfully), I think they belong better *here*, and not *there*.